packages/entities/entities-shared/src/components/entity-base-config-card/ConfigCardItem.vue

```diff
@@ -131,6 +143,9 @@ import InternalLinkItem from './InternalLinkItem.vue'
 import StatusBadge from './StatusBadge.vue'
 import '@kong-ui-public/copy-uuid/dist/style.css'
 
+// 20ch + ellipsis(3ch) + 1
+const MAX_LABEL_LENGTH = 24
```

Comment:
This may not be accurate. Because we are not using a monospaced font, different characters will have very different widths, e.g. 'l' and 'W'. A better way would be to render a `<span>` with the same style off-screen and check its `clientWidth` and `scrollWidth`.

That being said, I can accept this as a temporary solution.

Reply:
Would it be helpful to modify KTooltip so that it does not render the popup if the `label` is empty?

Reply:
> Would it be helpful to modify KTooltip so that it does not render the popup if the label is undefined?
Yes, it will be helpful.

If KTooltip does not render the popup when `label` is undefined, we still need to know when the text is being truncated.

Reply:
Big +1 that it would be nice to not render the tooltip if `label` is `undefined`. We end up writing conditional code to support this scenario A LOT.

Reply:
Updated
https://github.com/Kong/kongponents/pull/1783